python/timeplus/dbapi.py

```diff
@@ -151,20 +151,27 @@ def __exit__(self, *exc):
     # since some SQL is not exposed through API yet
     def _exist_table(self, name):
         stream = Stream(env=self.env).name(name)
-        return stream.exist()
+        external_stream = ExternalStream(env=self.env).name(name)
+        return stream.exist() or external_stream.exist()
 
     def _exist_view(self, name):
         view = View(env=self.env).name(name)
         return view.exist()
 
     def _get_table(self, name):
-        return Stream(env=self.env).name(name).get()
+        try:
+            return Stream(env=self.env).name(name).get()
+        except:
+            return ExternalStream(env=self.env).name(name).get()
 
     def _get_view(self, name):
         return View(env=self.env).name(name).get()
 
     def _list_table(self):
-        return Stream(env=self.env).list()
+        streams =  Stream(env=self.env).list()
+        estreams = ExternalStream(env=self.env).list()
```

Comment:
do we want to list external table?

Reply:
Yes, that is good to think about, so far the python SDK did not include external table yet, I probably will add it in next PR